public/static/docs/command-reference/metrics/diff.md

```diff
@@ -0,0 +1,123 @@
+# metrics diff
+
+Show a table of changes in [metrics](/doc/command-reference/metrics#description)
+between two versions of the <abbr>DVC repository</abbr>.
```

Comment:
> From https://github.com/iterative/dvc.org/pull/933/files/5fbec225702734a77b8d532df587c32db380ce15#r378634225

On the other hand it compares _workspace_ and the last commit - means that it's not always a commit/commit comparison. Here's how git handles it:

`git-diff - Show changes between commits, commit and working tree, etc`

Reply:
You are correct again @shcheklein . Check my latest revision:

`Show changes in metrics between commits in the DVC repository, or between a commit and the workspace.`

(similarly applied to `dvc diff` in 1473ebe1 and to the core repo in https://github.com/iterative/dvc/pull/3308/commits/d0ce5c4f497db07b8dd37b10d62fe34556a85689)

Reply:
👍